internal/vcs/git/refs_test.go

```diff
@@ -304,3 +304,122 @@ func TestRepository_ListTags(t *testing.T) {
 		}
 	}
 }
+
+func TestValidateBranchName(t *testing.T) {
+	for _, tc := range []struct {
+		name   string
+		branch string
+		valid  bool
+	}{
+		{
+			name:   "Valid branch",
+			branch: "valid-branch",
+			valid:  true,
+		},
```

Comment:
Another super nitpicky things that you can ignore, but ... since I don't have a vertical monitor setup, how about changing these test cases to this? 😎 

```suggestion
		{ name: "Valid branch", branch: "valid-branch", valid: true },
```